### DISCUSSION TITLE
Update the View code dynamically

### DISCUSSION THREAD

**Discussion by hardikdgsa** [GrapesJS/grapesjs#5913] [Q&A]
I am using GrapeJS for building the email templates. For the dynamic binding and templates. I am updating the text dynamically from the code only. These changes are reflected in the HTML preview/code; however, when I press View code these changes are not reflected. 

Which method or event should be called to update the changes to the view code that is in the current HTML page?

**Comment by artf** [comment]
Ensure you're editing the model of the component and not the view/DOM
